person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name: 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   def can_use_services?
```

Comment:
- [ ] As mentioned above for the book class, in the same way please make sure the person class also has the has many rentals association and a method whereby when we add rental for a person a rental is also created with the person as self parameter like :point_down: 
```
 def add_rental(book, date)
    Rental.new(date, book, self)
  end
```